src/Reference/Commitments/SFA.DAS.Payments.Reference.Commitments/ApiProcessor.cs

```diff
@@ -120,7 +120,9 @@ private AddOrUpdateCommitmentCommandRequest ConvertEventToCommand(Events.Api.Typ
                 VersionId = @event.Id,
                 Priority = @event.PaymentOrder,
                 PaymentStatus = paymentStatus,
-                LegalEntityName = @event.LegalEntityName
```

Comment:
`@event`  looks 🤢 
I know it's not your code, but worth changing.

Reply:
I've changed this and also added the EventPublisher project into the solution and updated its packages